webbpsf/trending.py

```diff
@@ -1,20 +1,19 @@
 import calendar
-import datetime
 import functools
 import os
 
 import astropy
 import astropy.io.fits as fits
 import astropy.time
 import astropy.units as u
-from astroquery.mast import Observations
 import matplotlib
 import matplotlib.pyplot as plt
-from matplotlib.backends.backend_pdf import PdfPages
 import numpy as np
+import poppy
 import scipy.interpolate
+from astroquery.mast import Observations
+from matplotlib.backends.backend_pdf import PdfPages
```

Comment:
super minor comment/question: should this `from matplotlib` import be a couple lines higher up, grouped with the other matplotlib imports?

Reply:
I believe the best practice is all direct `import` statements are grouped above all `from ... import ....` statements